Algorithms_Concepts/BubbleSort.cs

```diff
@@ -0,0 +1,49 @@
+//program for bubble sort 
+//Bubble Sort Ascending order
+using System;
+namespace Algorithms_Concepts
+{
+    public class BubbleSort
+    {
+        int arrayLoop;
+        int arraySize;
+        int[] array = new int[100];
+        public void AcceptArray()
+        {
+               Console.WriteLine ("Define Array size");
+               arraySize = Convert.ToInt32 (Console.ReadLine ());
+             
+               for (arrayLoop = 0; arrayLoop < arraySize; arrayLoop++)
+               {
+                   Console.WriteLine ("Enter Value");
+                   array[arrayLoop] = Convert.ToInt32 (Console.ReadLine ());
+               }
+        }
+        public void SortArray()
+        {
+            
+
+           for (arrayLoop = 0; arrayLoop < arraySize; arrayLoop++) 
+           {
+              for (int inetrnalLoop = 0; inetrnalLoop < arraySize - 1; inetrnalLoop++) 
```

Comment:
typo, also rename inetrnalLoop -> 'inner' and the above loop variable to 'outer'